### PR TITLE
Preparing release 0.8.2

### PR DESCRIPTION
**Pull request by Natim**
Fixing the heartbeat bug found in production.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ install:
     - .venv/bin/pip install uwsgi
 before_script:
     - cp $TRAVIS_BUILD_DIR/app.wsgi $HOME/
-    - cd $HOME && KINTO_INI=$TRAVIS_BUILD_DIR/config/kinto.ini UWSGI_VIRTUALENV=$TRAVIS_BUILD_DIR/.venv $TRAVIS_BUILD_DIR/.venv/bin/uwsgi --http :8888 $TRAVIS_BUILD_DIR/config/kinto.ini &
+    - cd $HOME && KINTO_INI=$TRAVIS_BUILD_DIR/config/kinto.ini $TRAVIS_BUILD_DIR/.venv/bin/pserve $TRAVIS_BUILD_DIR/config/kinto.ini &
```
Comment (leplatrem) [inline .travis.yml]: Does this fix #50 ?

**Comment by leplatrem** [comment]
r+

File: .travis.yml (hunk 1 above)
Comment (Natim) [inline .travis.yml]: The issue was created when we did that change on master.
We might want to get back uwsgi in the play with a less greedy conf.

**Pull request merged**